### PR TITLE
login_page

### PR DESCRIPTION
**Pull request by Bohdan-Fursa**

### CODE REVIEW

File: zoo-hope/src/index.tsx [hunk 1]
```diff
@@ -1,23 +1,20 @@
...
 
 root.render(
   <React.StrictMode>
-    <BrowserRouter>
-      {true ? 
-        <App /> :
-        <AdminPage />
-      }
-    </BrowserRouter>
+    {/* <BrowserRouter>{true ? <App /> : <AdminPage />}</BrowserRouter> */}
```
Comment (Osttik) [inline zoo-hope/src/index.tsx]: Uncomment that

File: zoo-hope/src/index.tsx [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
+    <LoginPage />
```
Comment (Osttik) [inline zoo-hope/src/index.tsx]: For now remove that, we will thing in future, where we will add that

**Review by Osttik — CHANGES_REQUESTED** [review]

File: zoo-hope/src/index.tsx (hunk 2 above)
Comment (Bohdan-Fursa) [reply zoo-hope/src/index.tsx]: Done

File: zoo-hope/src/index.tsx (hunk 1 above)
Comment (Bohdan-Fursa) [reply zoo-hope/src/index.tsx]: Done

**Pull request merged**